packages/tanstack-solid-query/test/__helpers.tsx

```
@@ -0,0 +1,55 @@
+import '@testing-library/dom';
+import '@testing-library/jest-dom/vitest';
+import type { TestServerAndClientResourceOpts } from '@trpc/client/__tests__/testClientResource';
+import { testServerAndClientResource } from '@trpc/client/__tests__/testClientResource';
+import { render } from '@solidjs/testing-library';
+import { QueryClient, QueryClientProvider } from '@tanstack/solid-query';
+import { getUntypedClient } from '@trpc/client';
+import type { AnyTRPCRouter } from '@trpc/server';
+import { JSX } from 'solid-js/jsx-runtime';
```

ESLint

[error] All imports in the declaration are only used as types. Use `import type`.

(@typescript-eslint/consistent-type-imports)